.travis.yml

```diff
@@ -8,18 +8,13 @@ matrix:
   include:
   - os: linux
     dist: trusty # Ubuntu 14.04
-    dotnet: 1.0.0-preview2-1-003177
+    dotnet: 1.0.1
     mono: none
     env: DOTNETCORE=1
     sudo: required
-  - os: osx
```

Comment:
I removed it 'cause I thought you meant this is what was breaking it. ok, i'll add it back and try and make it work, if I fail, i'll remove it again because i think it's better to get the green light and have this as an open todo.

i've got it building now anyway.